lib/utils/helpers.ts

```diff
@@ -445,6 +445,18 @@ export const expandForAppName = {
 	},
 } as const;
 
+export const devicesSelectFields = {
```

Comment:
@thgreasi if we make `const fields = ...` as you suggested, this is a breaking change because the order of the output actually changes (for example, the device_type property should be the fourth element on the array for output not change....